src/test/java/com/virtuslab/archunit/UiThreadUnsafeMethodInvocationsTestSuite.java

```diff
@@ -101,54 +155,19 @@ public void check(JavaMethod method, ConditionEvents events) {
-                    "git4idea.branch.GitNewBranchDialog.showAndGetOptions()",
-                    "git4idea.branch.GitNewBranchOptions.getName()",
-                    "git4idea.branch.GitNewBranchOptions.shouldCheckout()",
-                    "git4idea.config.GitSharedSettings.getInstance(com.intellij.openapi.project.Project)",
-                    "git4idea.config.GitSharedSettings.isBranchProtected(java.lang.String)",
-                    "git4idea.config.GitVcsSettings.getInstance(com.intellij.openapi.project.Project)",
-                    "git4idea.config.GitVcsSettings.getRecentRootPath()",
-                    "git4idea.fetch.GitFetchSupport.isFetchRunning()",
-                    "git4idea.fetch.GitFetchResult.showNotification()",
-                    "git4idea.fetch.GitFetchSupport.fetchSupport(com.intellij.openapi.project.Project)",
-                    "git4idea.push.GitPushSource.create(git4idea.GitLocalBranch)",
-                    "git4idea.repo.GitRemote.getName()",
-                    "git4idea.repo.GitRepository.getBranches()",
-                    "git4idea.repo.GitRepository.getCurrentBranch()",
-                    "git4idea.repo.GitRepository.getProject()",
-                    "git4idea.repo.GitRepository.getRemotes()",
-                    "git4idea.repo.GitRepository.getRoot()",
-                    "git4idea.repo.GitRepository.getState()",
-                    "git4idea.repo.GitRepository.getVcs()",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.<init>(javax.swing.ComboBoxModel, com.intellij.openapi.project.Project)",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.addDocumentListener(com.intellij.openapi.editor.event.DocumentListener)",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.getModel()",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.getText()",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.setPlaceholder(java.lang.String)",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.setPrototypeDisplayValue(java.lang.Object)",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.setUI(javax.swing.plaf.ComboBoxUI)",
-                    "git4idea.ui.ComboBoxWithAutoCompletion.selectAll()",
-                    "git4idea.validators.GitBranchValidatorKt.checkRefName(java.lang.String)"
-                };
-                String calledMethodFullName = callTarget.getFullName();
-
-                if (!Arrays.asList(whitelistedMethodFullNames).contains(calledMethodFullName)) {
+                if (!Arrays.asList(whitelistedMethodFullNames_git4idea).contains(calledMethodFullName)) {
                   String message = "a non-${UIThreadUnsafeName} method ${method.getFullName()} " +
                       "calls method ${calledMethodFullName} from git4idea";
```

Comment:
so soon the plugin name will be vcs-git but the package will remain git4diea, right?

Reply:
:yes:, exactly